programs/validator-history/src/lib.rs

```diff
@@ -94,4 +106,16 @@ pub mod validator_history {
     pub fn copy_gossip_contact_info(ctx: Context<CopyGossipContactInfo>) -> Result<()> {
         instructions::copy_gossip_contact_info::handler(ctx)
     }
+
+    pub fn copy_cluster_info(ctx: Context<CopyClusterInfo>) -> Result<()> {
+        instructions::copy_cluster_info::handler(ctx)
+    }
+
+    pub fn backfill_total_blocks(
```

Comment:
backfill_cluster_history?

Reply:
as of now it's just the total blocks field, but either way looking to remove this method after we run the backfill so not too worried about what it's named